CHANGELOG.md

```diff
@@ -1,222 +1,304 @@
-# Change Log for hipBLAS
-
-## hipBLAS 2.0.0 for ROCm 6.0.0
-### Added
-- added option to define HIPBLAS_USE_HIP_BFLOAT16 to switch API to use hip_bfloat16 type
-- added hipblasGemmExWithFlags API
-### Deprecated
-- hipblasDatatype_t is deprecated and will be removed in a future release and replaced with hipDataType
-- hipblasComplex and hipblasDoubleComplex are deprecated and will be removed in a future release and replaced with hipComplex and hipDoubleComplex
-- use of hipblasDatatype_t for hipblasGemmEx for compute-type is deprecated and will be replaced with hipblasComputeType_t in a future release
-### Removed
-- hipblasXtrmm that calculates B <- alpha * op(A) * B is removed and replaced with hipblasXtrmm that calculates C <- alpha * op(A) * B
+# Changelog for hipBLAS
+
+Documentation for hipBLAS is available at
+[https://rocm.docs.amd.com/projects/hipBLAS/en/latest/](https://rocm.docs.amd.com/projects/hipBLAS/en/latest/).
+
+## (Unreleased) hipBLAS 2.0.0 for ROCm 6.0.0
```

Comment:
This can just be `## hipBLAS 2.0.0 for ROCm 6.0.0` as it was updated with #714, sorry for the merge conflicts.
Otherwise looks good! After this change you can go ahead and merge, or I can if there's permission issues.